translations/town.tex/pl.po

```diff
@@ -80,7 +80,21 @@ msgid ""
 "Some Building Tiles are double-sided, and may later be upgraded and flipped to represent two different buildings at the same time. Such upgrades must be \\textbf{Built in order}.\\par\n"
 "\n"
 msgstr ""
-
+"\\hypertarget{Miasto}{Każda}\\index{Miasto} frakcja ma własne miasto, które znajduje się na środku swojego kafla początkowego.\n"
+"Miasto jest twoją najważniejszą lokacją - wiele scenariuszy może się \\hyperlink{End}{zakończyć}, jeśli \\hyperlink{Categories}{oflaguje} je bohater wroga.\\par\n"
```

Comment:
Mimo wszystko Lokacje też dałbym wielką literą - to element gry

```suggestion
"Miasto jest twoją najważniejszą Lokacją - wiele Scenariuszy może się \\hyperlink{End}{zakończyć}, jeśli \\hyperlink{Categories}{oflaguje} je Bohater wroga.\\par\n"
```

Reply:
Zamiast `Categories` proszę dać `Typy Lokacji`.

Reply:
A zamiast `End` - `Koniec`.